examples/reminders/lib/base/data_sources/remote/reminders_firebase_data_source.dart

```diff
@@ -2,6 +2,7 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:firebase_auth/firebase_auth.dart';
```

Comment:
@StanislavPrime, could you please review the changes in this file ?

Reply:
@StanevPrime I think that everything is working as expected. I tested to login with a new user, for which there were no previous collections in firebase and saw that the counters and reminders length collections are not created any more and the methods for them from the data source were deleted. So these collections are not needed any more and they can be deleted from each of the 3 firebase projects.